mpa/modules/models/detectors/unbiased_teacher.py

```diff
@@ -128,9 +128,45 @@ def forward_train(self,
                 loss*self.unlabeled_loss_weight for loss in ul_loss
             ]
         # TODO: apply loss_bbox when adopting QFL;
+        # For debug
+        # self.save_pseudo_labels(ul_img0, pseudo_bboxes, pseudo_labels)
 
         return losses
 
+    def save_pseudo_labels(self, ul_imgs, pseudo_bboxes, pseudo_labels):
+        """This saves unlabeled image with pseudo bboxes, this is for debug."""
+        import os
+        import cv2 as cv
+
+        self.save_dir = "/home/jaeguk/workspace/tmp/semisl-det/"
```

Comment:
I want you to revise this save_dir path if you wanna remain this save_pseudo_labels methods.

Reply:
Change the path to /tmp/ Thanks for review!